src/app/search/filters.html

```diff
@@ -300,14 +300,11 @@ <h1 ng-i18next="Header.Filters"></h1>
       </span>
 
       <tr>
-        <td><span>is:year1</span> <span>is:year2</span> <span>is:year3</span></td>
+        <td><span>year:=2</span></td>
```

Comment:
You don't have to type `=` - `year:1` will also work.

Reply:
fixed